Resources/Private/Fusion/Prototypes/Select.fusion

```diff
@@ -1,21 +1,36 @@
 prototype(Neos.Fusion.Form:Select)  < prototype(Neos.Fusion.Form:Field) {
+    @context.__multiple = ${this.multiple}
 
     @propTypes {
         content = ${PropTypes.string}
+        multiple = ${PropTypes.boolean}
     }
 
+    content = null
+    multiple = false
+
     renderer = afx`
         <select
             {...props.attributes}
-            type="text"
-            name={field.name}
+            id={props.id}
+            class={props.class}
+            name={field.name + (props.multiple ? '[]' : '')}
+            required={props.required}
+            multiple={props.multiple}
         >
             {props.content}
         </select>
     `
 
-    // check against field.value as for automatic selected state
     prototype(Neos.Fusion.Form:Select.Option) {
-        renderer.attributes.selected = ${(field && (Form.stringifyValue(field.value) == Form.stringifyValue(props.value))) ? true : false}
+        # override selected for object/submitted values
+        renderer.attributes.selected.@process.singlevalue = ${(field.current == props.value) ? true : false}
+        renderer.attributes.selected.@process.singlevalue.@if.hasCurrent = ${field.current}
+        renderer.attributes.selected.@process.singlevalue.@if.isSingle = ${!__multiple}
+
+        # override selected for object/submitted multivalues
+        renderer.attributes.selected.@process.multivalue = ${Array.indexOf(field.current, props.value) > -1}
+        renderer.attributes.selected.@process.multivalue.@if.hasCurrent = ${field.current}
+        renderer.attributes.selected.@process.multivalue.@if.isMultiple = ${__multiple}
```

Comment:
should we maybe write the Single/Multivalue distinction as Neos.Fusion:Case instead of "abusing" processors here?

Reply:
Switched to wrapper components now